lib/common/xml.c

```diff
@@ -328,32 +328,31 @@ pcmk__xml_position(xmlNode *xml, enum xml_private_flags ignore_if_set)
     return position;
 }
 
-// Remove all attributes marked as deleted from an XML node
-static void
-accept_attr_deletions(xmlNode *xml)
+// This also clears attribute's flags if not marked as deleted
+static bool
+marked_as_deleted(xmlAttrPtr a, void *user_data)
```

Comment:
well the comment is there but wouldn't we be on the safe side to reflect modification in the naming here?
not sure if 'marked_as_deleted_otherwise_clear' is especially beautiful ...

Reply:
I can't think of another name that's not confusing